Developed main page, controller and functional test

ProgrammSM opened the pull request.

Commit:
Заготавливаем главную страницу и вычисление

Commit:
Создаём главную страницу + тесты

Исправляем SiteController, изменяем представление index, добавляем функциональный тест IndexCest, исключаем предыдущие тесты

Commit:
Исправляем в соответствии с задачей

Commit:
Удаляем лишнее из данной задачи

Commit:
Исправляем ошибку с SiteController

Исправлена ошибка обрушивающая тесты

Commit:
Исправляем представление index.php

artezh commented on controllers/SiteController.php:
```diff
@@ -2,61 +2,20 @@
 namespace app\controllers;
 
-use app\models\test;
+//use app\models\CalcForm;
```
Необходимо удалить

artezh commented on controllers/SiteController.php:
```diff
@@ -2,61 +2,20 @@
-use yii\filters\VerbFilter;
-use app\models\LoginForm;
-use app\models\ContactForm;
+//use yii\web\UploadedFile;
```
удалить комент

artezh commented on controllers/SiteController.php:
```diff
@@ -2,61 +2,20 @@
-
-    /**
-     * Displays homepage.
+     * Отображаем главную сраницу.
```
сраница

artezh commented on controllers/SiteController.php:
```diff
@@ -65,63 +24,33 @@ public function actionIndex()
-    {
-        return $this->render('about');
-    }
+//    /**
```
Зачем это всё закоментировано?

artezh commented on tests/functional/IndexCest.php:
```diff
@@ -0,0 +1,44 @@
+        $I->see('Калькулятор работ', 'h1');
+    }
+
+//    public function goingOnNavToCalcPageAndBack(\FunctionalTester $I)
```
то же самое, если не нужно - удаляем

Commit:
Чистим код, исправляем опечатки

Исправляем опечатку в SiteController и удаляем ненужные комментарии из SiteController и IndexCest

artezh commented on views/layouts/main.php:
```diff
@@ -66,10 +53,8 @@
 </div>
 
```
пустые линии зачем?

Commit:
Удаляем пустые строки
